### DISCUSSION TITLE
How to configure namespace in web3modal?

### DISCUSSION THREAD

**Discussion by CjoyHome** [WalletConnect/walletconnect-monorepo#4023] [Developer Support] [answered]
version：
```
  "@walletconnect/modal": "^2.6.2",
    "@web3modal/ethers5": "^3.4.0",
    "ethers": "^5.7.2"
```

<img width="908" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/26813427/b88b7235-bcd2-4d81-990f-26ce36c8d701">
<img width="645" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/26813427/20cc07b7-82f8-46af-be94-ed3ac3586d3a">

I need to connect to the safe wallet. I can't find the relevant method of configuring namespaces in the web3modal tutorial. I hope I can get help.

**Comment by glitch-txs** [comment, marked as answer]
Hi, it's done already internally in ethers :)
